detekt/wrapper/src/test/java/io/buildfoundation/bazel/detekt/ExecutionUtilsTests.java

```diff
@@ -0,0 +1,74 @@
+package io.buildfoundation.bazel.detekt;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class ExecutionUtilsTests {
+
+    @Test
+    public void testShouldRunAsTestTargetWhenTrue() {
```

Comment:
```suggestion
    public void shouldRunAsTestTargetWhenTrue() {
```

No need for `test` prefix in the method names here and below :)